website/pages/blog/open-source-graphql-security.mdx

```diff
@@ -0,0 +1,249 @@
+potential threats. Open source and public tools have been doing a great job at protecting it for
+years.
+
+Cybersecurity thrives on collaboration, exemplified by the Common Vulnerabilities and Exposures
+([CVE](https://cve.mitre.org/)) system. This public database acts as a central repository for known
+cybersecurity threats, allowing for quick dissemination and response. By making vulnerabilities
+public, the CVE system ensures timely and widespread implementation of security measures. For
+instance, the identification and patching of the Heartbleed bug in OpenSSL was significantly aided
+by the CVE system, showcasing its effectiveness in promoting rapid response.
+
+Open source plays a crucial role in cybersecurity by fostering an environment of transparency and
+collaboration. It allows for the open identification and resolution of vulnerabilities, benefiting
+the entire digital ecosystem. For example, the Linux Kernel, known for its security, continually
+improves through community contributions. Similarly, tools like Kali Linux offer insights into
+offensive cybersecurity strategies, helping developers strengthen their defenses.
+
+This combination of open source and CVE is especially vital in areas like GraphQL and API security,
+providing a foundation for robust, adaptable cybersecurity strategies. Through shared knowledge and
+tools, open source and CVE create a proactive defense against evolving cyber threats.
+
+In a field like cybersecurity, where dangers morph quickly, having shared knowledge, united
+alertness, and available-to-all tools become an essential shield against potential attacks. The
+marriage of open source and cybersecurity offers a lively platform for ongoing learning, changing,
+and sharing attack and defense mechanisms, crafting a global, united front against cyber threats.
+
+Moving forward, we'll dive into how this perfect pairing of open source and cybersecurity is
+crucial, not just relevant, for GraphQL and API security. We'll highlight practical tools and
+strategies you can use today to protect your applications.
+
+## Open Source GraphQL Security Ressources & Tools
+
+There are many GraphQL open-source tools available to help developers and businesses defend against
+possible cybersecurity threats. From defensive measures that shield sensitive data to offensive
+tools aimed at identifying vulnerabilities, the open-source community has build invaluable resources
+to cover a wide variedy of cyber-security needs.
+
+### Learning Tools & Resources: Armoring with Knowledge
+
+In the sphere of cybersecurity, especially concerning GraphQL, the adage 'knowledge is power' is
+paramount. Continual learning, embracing best practices, and leveraging insights from the community
+is an essential shield to secure APIs against vulnerabilities.
+
+1.  **Best Practices**
+
+    - [Automatic persisted queries](https://the-guild.dev/graphql/yoga-server/docs/features/automatic-persisted-queries)
```

Comment:
I think here it suppose to be persisted queries (or in their new name "trusted queries") and not automatic persisted queries, which are actually not secure?

Reply:
Yes, you're right sorry for the confusion. Do you have any docs / tech in mind to link here, I'm not an expert of persisted queries and I don't know the references to be honest. There doesn't seem to be a standard / easy way to do it as of now?

Reply:
I would like here: https://benjie.dev/graphql/trusted-documents and here: https://the-guild.dev/graphql/yoga-server/docs/features/persisted-operations